page/index.html

```diff
@@ -1,99 +1,158 @@
+		</div>
 		<figure>
 			<img src="../images/bullets.png" alt="">
 		</figure>
 		<ul>
-			<li><div class="bullet-larg-text">.223</div> REMINGTON 69gr BTHP</li>
-			<li><div class="bullet-larg-text">.308</div> WINCHESTER 168gr BTHP</li>
-			<li>7.62x39mm RUSSIAN 154gr JSP</li>
-			<li><div class="bullet-larg-text">7.62</div> THUMPER 220gr RN</li>
-			<li><div class="bullet-larg-text">7.62</div> THUMPER 240gr BTHP</li>
+			<li class="bullet-one">
+				<div class="bullet-med-text">.223</div><span class="brand-small">REMINGTON</br></span><span class="bullet-small-text">69gr BTHP<span></li>
+			<li class="bullet-two"><div class="bullet-med-text">.308</div><span class="brand-small">WINCHESTER</br></span><span class="bullet-small-text">168gr BTHP<span></li>
+			<li class="bullet-three"><div class="bullet-small-text">7.62x39mm</div><div class="large-list">RUSSIAN 154gr JSP</div></li>
+			<li class="bullet-four"><div class="bullet-large-text">7.62</div><div class="large-list">THUMPER  220gr RN</div></li>
+			<li class="bullet-five"><div class="bullet-large-text">7.62</div><div class="large-list">THUMPER</div><span class="bullet-small-text">240gr  BTHP<span></li>
 		</ul>
 	</article>
-
-	<article>
+	<article class="displaying-blocks">
 		<h1>DISPLAYING (FAMOUS) BLOCKS</h1>
 		<figure>
-			<img src="../images/abbey.png" alt="">
+			<img class="abbeyroad" src="../images/abbey.jpg" alt="">
+		</figure>
+		<figure>
+			<img class="map" src="../images/map.png" alt="">
 		</figure>
 		<section>Abbey Road is a thoroughfare lcoated in the borough of Camden and the City of Westminster in London, running roughly northwest to southeast through St. John's Wood, near Lord's Cricket Ground.  It is part of the B507.  This road is best known fo the Abbey Road Studios and the 1969 album, Abbey Road, by The Beatles.</section>
 		<aside>Photo goo.gl/h44dc9</aside>
 	</article>
-	<article>
+	<article class="border-bottom-line">
 		<figure>
-			<img src="../images/heads.png" alt="">
+			<img src="../images/heads.jpg" alt="">
 		</figure>
 		<h1>BORDER BOTTOM LINE</h1>
 		<h3>Awful immigration laws from around the world.</h3>
 		<section><strong> Italy </strong> penalizes illegal immigrants with a fine of &#8364 5,000-10,000 and allows immigration officials to detain them for up to 6 months. <strong> &#8226 Switzerland </strong> immediately deports all convicted criminals from other countries andpotentially even their family members. <strong> &#8226 Australia</strong> detains all non-citizens without a valid visa. <strong>&#8226 Japan</strong> is allowed to pay $,3000 to each unemployed Latin American immigrant of Japanese descent and $2,000 to each of that unemployed worker's family members to return to their country of origin. &#8226 The <strong>United Arab Emirates</strong> prohibits foreigners form engaging in any sort of labor union--like activy, resulting in harsh living conditions, 80-hour work weeks, back-breaking manual labor, and below-minimum-wage pay.</section>
```

Comment:
Section lacks heading. Character reference was not terminated by a semicolon, `&#8364`